advisor/backend.go

```diff
@@ -206,3 +227,25 @@ func (f *boltFinder) Find(command string) ([]Command, error) {
 
 	return cmds, nil
 }
+
+func (f *boltFinder) FindPackage(pkgName string) ([]Package, error) {
+	tx, err := f.Begin(false)
+	if err != nil {
+		return nil, err
+	}
+	defer tx.Rollback()
+
+	b := tx.Bucket(pkgBucketKey)
+	if b == nil {
+		return nil, nil
+	}
+
+	bsummary := b.Get([]byte(pkgName))
+	if bsummary == nil {
+		return nil, nil
+	}
+
+	pkg := Package{Snap: pkgName, Summary: string(bsummary)}
+
+	return []Package{pkg}, nil
```

Comment:
The function is declared to return a list of pacakges, but in fact it returns only one. Is there a plan to do partial matches on package name?

Reply:
I’ll add a small request to just document the function as a part of the answer

Reply:
Good point, this is for exact matches only, fixed now.